src/getter.js

```diff
@@ -102,6 +102,35 @@ function getStoreDeps(getter) {
   return storeDeps
 }
 
+/**
+ * Add override options to a getter
+ * @param {getter} getter
+ * @param {object} options
+ * @param {boolean} options.useCache
+ * @param {*} options.cacheKey
+ * @returns {getter}
+ */
+function addGetterOptions(getter, options) {
```

Comment:
This API change to introduce the concept of "getter options" is a little awkward imo.

Currently, it will replace any previously set options which could lead to unexpected behavior given its name.
An alternative would be to call this `setGetterOptions` to indicate it would replace, not patch, the options.
But I feel like the API is being conflated with implementation details of hidden `__options` object. The alternative that I would suggest is two separate methods to configure these options.

cc @jordangarcia since this is API design considerations